osrdyne/src/drivers/docker.rs

```diff
@@ -47,11 +50,17 @@ impl DockerDriver {
         amqp_uri: String,
         worker_pool: String,
     ) -> DockerDriver {
+        let version_identifier = std::env::var("OSRD_GIT_DESCRIBE")
+            .unwrap_or_else(|_| format!("run-{}", Uuid::new_v4()));
```

Comment:
This should work for now, and this PR is time critical, so feel free to ignore it.

I'm not a fan of relying on an env var for this. Maybe we should embed a generated uuid at build time?

Reply:
Noted, we will change this in a future PR.